### PR TITLE
Internal changes to support tox-gh-actions

### PR DESCRIPTION
**Pull request by gaborbernat**
For https://github.com/gaborbernat/tox-gh-actions/blob/support-v4/src/tox_gh_actions/plugin.py#L1-L108

### CODE REVIEW

File: src/tox/plugin/manager.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 from tox.tox_env.python.virtual_env.package import api
 from tox.tox_env.register import REGISTER, ToxEnvRegister
 
+from ..config.main import Config
```
Comment (jugmac00) [inline src/tox/plugin/manager.py:18]: Why not `from tox.config.main import Config`?

**Review by jugmac00 — APPROVED** [review]
Any chance to get the uncovered lines under test?

How did you test this? Did you actually run a modified version of `tox-gh-actions` with your branch?

Is it necessary that `tox-gh-actions` perform any changes or is it already 100% compatible with branch of tox4?

File: src/tox/plugin/manager.py (hunk 1 above)
Comment (gaborbernat) [reply src/tox/plugin/manager.py:18]: Prefer relative imports where possible :thinking:

**Comment by gaborbernat** [comment]
> Any chance to get the uncovered lines under test?

Yes :blush: 
 
> How did you test this? Did you actually run a modified version of `tox-gh-actions` with your branch?

Yes, locally.

> Is it necessary that `tox-gh-actions` perform any changes or is it already 100% compatible with this branch of tox4?

No, it's already compatible.

**Review by jugmac00 — APPROVED** [review]

**Pull request merged**